style.css

```diff
@@ -0,0 +1,473 @@
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.social-media {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  list-style-type: none;
+  margin-top: 10%;
+  margin-left: 10%;
+  width: 50%;
+}
+
+/* my recent wor */
+
+.recent-profesional {
+  background: #e5e5e5;
+  margin-left: 6.4%;
+  margin-right: 6.4%;
+  width: 87.2%;
+}
+
+.gridd {
+  display: grid;
+  grid-template-columns: auto;
+}
+
+.my-recent-works {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  background-color: white;
+  flex: 1;
+  height: 100vh;
+  width: 100vw;
+  padding-bottom: 10%;
+}
+
+.H1 {
+  position: relative;
+  width: 100%;
+  margin-top: 15%;
+  margin-left: 10%;
+  font-size: 24px;
+  align-self: center;
+  color: #1c1a19;
+}
+
+.line {
+  height: auto;
+  display: flex;
+  position: relative;
+  margin-left: 10%;
+  margin-right: 10%;
+  width: 80%;
+  margin-top: 2.5%;
+  align-self: center;
+}
+
+.main-image {
+  display: flex;
+  margin-top: 15%;
+  width: 100%;
+  text-align: center;
+}
+
+.H2 {
+  position: relative;
+  width: 90%;
+  margin-top: 5%;
+  margin-left: 10%;
+  font-size: 18px;
+  color: #1c1a19;
+  order: 0;
+}
+
+.p-recent {
+  position: relative;
+  width: 80%;
+  margin-left: 10%;
+  margin-right: 10%;
+  flex: 1;
+  overflow: hidden;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.buttons-0 {
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+  list-style-type: none;
+  margin-top: 5%;
+  margin-left: 10%;
+  margin-right: 10%;
+  width: 100%;
+  flex-wrap: wrap;
+}
+
+.buttons-0 > li > button {
+  border: 1px solid #8993a4;
+  padding: 10px;
+}
+
+.css0 {
+  margin-left: 10%;
+}
+
+.html0 {
+  margin-left: 20%;
+}
+
+.bootstrap0 {
+  margin-left: 30%;
+}
+
+.ruby0 {
+  margin-left: 70%;
+}
+
+.see-project {
+  position: relative;
+  margin-top: 20px;
+  width: 40%;
+  color: white;
+  background-color: #ff6b00;
+  height: 30px;
+  margin-bottom: 0;
+  margin-left: 12px;
+  align-items: center;
+}
+
+.see-project > a {
+  margin-left: 20%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.see-project > a :hover {
+  border: 2px solid blue;
+  text-decoration: underline;
+  text-decoration-color: blue;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.btn:hover {
+  border: 2px solid blue;
+  text-decoration: underline;
+  text-decoration-color: blue;
+}
+
+/* profesional-art */
+
+.profesional-art-1 {
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  justify-content: center;
+  position: relative;
+  margin-left: 20px;
+  width: 100vw;
+  height: 40vh;
+  background-image: url(image/background.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
```

Comment:
- [ ] [REQUIRED] Please make changes!
- 1-The `<h2 />` part of the card is embedded in the text in <p /> and there are errors in the location and colors of your buttons, please correct them according to the Figma design.

- 2-In this part, there is an error with margin or padding because in your project, the card stuck to the right side and as a result, a gap appeared on the left side. Please fix this issue!

- 3-Please fix the "See Project" `button` because they are different from the button given in Figma design.

![image](https://user-images.githubusercontent.com/63915024/170607274-534bb536-ea7b-4a87-a232-4dab75ad4fd6.png)